tools/setup/emoji/build_emoji

```diff
@@ -239,6 +255,31 @@ def dump_emojis(cache_path):
             os.path.join(cache_emoji, fn)
         )
 
+    sprite_sheet_google = os.path.join(EMOJI_DATA_PATH, 'sheet_google_32.png')
+    run(['cp', sprite_sheet_google, cache_path], shell=True)
+    emoji_classes = []
+    emoji_class = ".emoji_%(codepoint)s"
```

Comment:
I'd maybe call this emoji_class_template (or similar), and similarlywith emoji_pos_info.